### ISSUE TITLE
Add signal about which DVC is being used

### ISSUE THREAD

**Issue by daavoo** [iterative/vscode-dvc#2925]
The extension supports multiple options for `Setup the Workspace`:  `Auto`, `Manual`, `Global`.

However, there is no signal to identify which option is being currently being used by the extension (beyond inspecting the `Output` of the extension where each call to dvc might include the full path to the venv).

I think the DVC extension could show a signal in the current status bar (i.e. extending display name from the current `DVC` to `DVC (Auto)`).

It might also be useful for the `Auto` and `Manual` options to display the name of the actual venv used similar ,
to how the Python extension does it:

<img width="269" alt="Captura de Pantalla 2022-12-13 a las 18 22 49" src="https://user-images.githubusercontent.com/12677733/207401634-ad3ae9ae-234c-47c9-831f-4f35c99aff2a.png">

**Comment by SoyGema** [comment]
as food for thought, this **signal** might have several forms:

1 ) A **downright popup** that includes the python path and version used by the extension and the virtual environment. 
2 ) The python version next to the **circle UI** in bottom left next to DVC icon .  

This could have an impact on DVCLive experiments integration. Python version is showed in python notebook and it´s recommendable to have an indication that is correctly synced with DVC.

Thanks for the time dedicated to this 👍

**Comment by mattseddon** [comment]
Semi-related.

As part of improving the onboarding experience, we are working on auto-installing DVC / DVCLive as Python packages if they cannot be located. 

@daavoo @SoyGema what are your initial thoughts on this screen:

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/37993418/208002954-7e445116-2b83-42ed-803f-308084b39c8c.png">

**Comment by SoyGema** [comment]
Looks great in my opinion! 

As a user , and taking into account left to right reading scenario, I would first **setup the workspace** and then **Select the python interpreter.**  Unclear to me if this is the happy path or it is the [other way around ](https://iterativeai.slack.com/archives/C01APS0FHDM/p1670842909689889?thread_ts=1670842047.540029&cid=C01APS0FHDM) as @daavoo mentioned . 



Regarding what happens once you setup both the **WorkSpace** and the Python Interpreter, It might be great to be shown something like this! 
<img width="1728" alt="HOW could BE" src="https://user-images.githubusercontent.com/24204714/208080036-080a8c95-135f-4e10-9813-97fad1464983.png">



**Regarding impact in DVCLive experiments :** 
Please Note that Jupyter Notebook makes visible at all times the python version and environment it is pointing too.  The setup has been my main roadblock during the exploration of the release.  Having also DVC setup more visible would be super useful .
<img width="1488" alt="Captura de Pantalla 2022-12-16 a las 11 22 09" src="https://user-images.githubusercontent.com/24204714/208081336-04fe221c-e6f4-4c05-97a2-dac8acc908fd.png">

Keep up the good work! 👍

**Comment by mattseddon** [comment]
Why would we duplicate the environment information in the UI if the Python extension is being used for virtual environment discovery? What problem are we trying to solve there?

**Comment by shcheklein** [comment]
I guess in case someone decided to manually specify, or global was detected for whatever reason. DVC is a superset in this sense to the Python extension.

**Comment by SoyGema** [comment]
Hello there team! thanks so much for implementing this.
Today I was dealing with a situation in which this verbose signal was useful : 

From a project, I created a branch in which only included the notebook for faster experiment tracking with live to test a hypothesis about changing the number of attention heads with langugage translation. I had to change from TF2.9 to TF2.10 as the implementation has changed.

DVC was installed globally, and I have to change it to a virtual environment that I was creating in that branch with another TF version . Also, I was able to see in the notebook that there was a correspondence in between the Python version from the notebook and DVC . 
<img width="1862" alt="Captura de Pantalla 2023-01-09 a las 13 15 03" src="https://user-images.githubusercontent.com/24204714/211309286-df378545-2f39-49c8-b72f-416a03c3be01.png">

This might be a minor thing, but workspace setup and correspondence verbosity has been super helpful! Thanks and congrats!
